benchmarks/.ruby-version

```diff
@@ -1 +1 @@
-3.0.0
```

Comment:
Should we remove this file? Or have a similar file in the root directory?

Reply:
What are implications of removing it?

Reply:
Discussed in person - let's remove this file since the gem supports multiple ruby versions